File: src/test/java/com/example/scaffold/usecases/CanaryTest.java
package com.example.scaffold.usecases;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit.jupiter.SpringExtension;

import static org.assertj.core.api.Assertions.assertThat;

@ExtendWith(SpringExtension.class)
@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
public class CanaryTest {
    @Test
    void should_be_able_to_run_spring_tests() {
        assertThat("Use-case test environment OK").isNotBlank();
    }
}

File: frameworks/context.core/src/test/kotlin/com/qomolangma/frameworks/context/core/CanaryTest.kt
package com.qomolangma.frameworks.context.core

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class CanaryTest {
    @Test
    fun should_be_able_to_run_tests() {
        assertThat("ok").isEqualTo("ok")
    }
}
